packages/web-components/src/components/va-file-input/test/va-file-input.e2e.ts

```diff
@@ -129,7 +22,7 @@ describe('va-file-input', () => {
     expect(hintTextElement.innerText).toContain('This is hint text');
   });
 
-  it('v3 renders a required span', async () => {
+  it('renders a required span', async () => {
     const page = await newE2EPage();
```

Comment:
Could the `uswds` attributes be removed now?

Reply:
Good find!, I thought I had removed all those.